_includes/0.4/left_sidebar.html

```diff
@@ -21,9 +21,9 @@
     </div>
     <div class="left-sidebar-group">
         <div class="left-sidebar-header">Splinter Concepts</div>
-        <a href="/docs/0.4/glossary/glossary.html">Splinter glossary</a>
-        <a href="/docs/0.4/concepts/about_canopy.html">About Canopy</a>
-        <a href="/docs/0.4/concepts/about_biome.html">About Biome</a>
+        <a href="/docs/0.4/glossary/glossary.html">Glossary</a>
```

Comment:
Did you intend to rename the glossary page from "Splinter Glossary" to just "Glossary"?

Reply:
Yes.  Two Splinters in a row seemed like overkill. 

Per doc best practices, it's OK to shorten the topic title for clarity in the left-nav TOC. Other entries already do this.

Reply:
That should be in the commit message than sense it currently looks like it could be a mistake

Reply:
Done.

Reply:
After further discussion on this change, I've removed it (and the TOC reorganization) so that this PR stays focused on renaming the "About" topics.